runtests.py

```diff
@@ -5,49 +5,57 @@
 #
 
 """
-File for running tests programmatically.
+Script for running Spyder tests programmatically.
 """
 
 # Standard library imports
 import os
 import sys
+import argparse
 
 # Third party imports
-import qtpy  # to ensure that Qt4 uses API v2
 import pytest
 
 
 # To run our slow tests only in our CIs
-run_slow = False
-if os.environ.get('CI', None) is not None or '--run-slow' in sys.argv:
-    run_slow = True
+RUN_CI = os.environ.get('CI', None) is not None
+run_slow = RUN_CI
 
-def main():
+
+def main(run_slow=run_slow, extra_args=None):
     """
-    Run pytest tests.
+    Run pytest tests for Spyder.
     """
     pytest_args = ['spyder',
                    'spyder_profiler',
-                   '-x',
                    '-vv',
                    '-rw',
-                   '--durations=10',
-                   '--cov=spyder',
-                   '--cov=spyder_profiler',
-                   '--cov-report=term-missing']
```

Comment:
Please restore everything of what you removed here, except for `'--cov-report=term-missing'`. That's still in master:

https://github.com/spyder-ide/spyder/blob/master/runtests.py#L41-L42

Reply:
I'm aware; this PR when merged would remove it there as well, per one of its major goals:

> Removes the remaining % file coverage by default, as it can easily be re-enabled locally in the case the user really desires it for every file, and is already displayed in far more accessible form via two different dedicated services when run the CIs and otherwise just adds extra clutter to the logs

However, if you have some practical use case for still keeping them in every CI report that isn't equally or better served by Codecov, Coveralls and running it locally, then I'm all ears.

Reply:
I don't have a problem with coverage being reported, but I'm almost never run the entire test suite locally, just the things I'm testing at the moment.

But sure, we can remove this.

Reply:
Okay, sure.